app/maildelay/urls.py

```diff
@@ -1,6 +1,6 @@
-from django.conf.urls import include, url
+from django.urls import include, re_path
 
 
 urlpatterns = [
-    url(r'^', include('mails.urls')),
+    re_path(r'^', include('mails.urls')),
```

Comment:
why the change to `re_path`?

Reply:
Found in [django documentation](https://docs.djangoproject.com/en/2.0/ref/urls/#url), that url is likely to be deprecated.

Reply:
Nice, TIL!